### PR TITLE
support for symfony 4

### PR DESCRIPTION
**Pull request by BorisCharriere**
add some modifications to be used in symfony 4 

see issue #48

### CODE REVIEW

**Comment by BorisCharriere** [comment]
@weaverryan there is one error concerning an old php version :"PHP 5.3 is supported only on Precise.
See https://docs.travis-ci.com/user/reference/trusty#PHP-images on how to test PHP 5.3 on Precise.
Terminating." What do we do for that ? 
the other error is about php 5.6 and '2.8.*@dev' the message is "PHP Fatal error:  Allowed memory size of 1610612736 bytes exhausted (tried to allocate 32 bytes) in phar:///home/travis/.phpenv/versions/5.6.32/bin/composer/src/Composer/DependencyResolver/Solver.php on line 220"

**Comment by weaverryan** [comment]
Nice job! Let’s only support php 5.5 and above. And also only support Symfony 2.8 and above. Let’s see if that fixes things :)

File: Resources/config/services.xml [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
+        <service id="knp_disqus.request" class="%knp_disqus.api.class%" public="true">
             <argument type="service" id="service_container" />
             <argument>%knp_disqus.api_key%</argument>
             <argument>%knp_disqus.secret_key%</argument>
             <argument>%knp_disqus.base_url%</argument>
             <argument>%kernel.debug%</argument>
         </service>
 
-        <service id="knp_disqus.helper" class="%knp_disqus.helper.class%">
+        <service id="knp_disqus.helper" class="%knp_disqus.helper.class%" public="true">
```
Comment (weaverryan) [inline Resources/config/services.xml]: I don’t think this service needs to be public

Comment (BorisCharriere) [reply Resources/config/services.xml]: helper class is used by twig function "knp_disqus_render"

**Comment by BorisCharriere** [comment]
as you can see the build failed due to a memory limit. And I dont't know how to fix it. Please have a look on that

**Comment by BorisCharriere** [comment]
@weaverryan  why don't we try withour @tag in travis ? maybe it causes problem

File: Resources/config/services.xml (hunk 1 above)
Comment (weaverryan) [reply Resources/config/services.xml]: You’re right :/

**Comment by weaverryan** [comment]
I'm not too worried about dev failing - I'll tag so we have a compat release. Thanks for your work!

**Pull request merged**